doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -167,8 +175,155 @@ The following security components are available:
+
+#. Click :guilabel:`Add file` in the **FILE** section, and select **Browse**.
+#. Navigate to where you extracted the HEX file and select it.
+#. Click the :guilabel:`Erase & write` button in the **DEVICE** section to program the DK.
+
+   Do not unplug or turn off the DK during this process.
+
+.. note::
+   If you experience any problems during the process, press ``Ctrl+R`` (``command+R`` on macOS) to restart the Programmer app, and try again.
+
+After you have programmed the sample to the DK, you can connect to it and test the functions.
+If you connect to the sample now, you can go directly to Step 2 of :ref:`nrf70_gs_connecting`.
+
+.. _nrf70_gs_connecting:
+
+Connecting to the sample
+************************
+
+You can connect to the sample on the nRF7002 DK with a terminal emulator on your computer using :term:`Universal Asynchronous Receiver/Transmitter (UART)`.
+This allows you to see the logging information the sample outputs.
+
+You can use an external UART to USB bridge.
+UART communication through the UART to USB CDC ACM bridge is referred to as CDC-UART.
+
+If you have problems connecting to the sample, restart the DK and start over.
+
+To connect using CDC-UART, complete the following steps:
+
+1. Connect the nRF7002 DK to the computer with a micro-USB cable, and then turn on the DK.
+
+   **LED5** starts blinking.
+
+#. Open a terminal emulator; this guide uses PuTTY.
+
+   The **PuTTY Configuration** window opens and the **Session** category is chosen in the category selection tree on the left.
+
+#. Select **Serial** as the **Connection type** under **Basic options for your PuTTY session**.
+
+   The text fields above the selection change to **Serial line** and **Speed**.
+
+#. Click the **Serial** category under the **Connection** category in the category selection tree to see options controlling the local serial line.
+#. Type the COM port corresponding to your DK in the **Serial line to connect to** field.
+
+   Depending on what devices you have connected to your computer, you might have several choices.
+   To find the correct port (in Windows):
+
+   a. Right-click on the Windows Start menu, and select **Device Manager**.
+   #. In the **Device Manager** window, scroll down and expand **Ports (COM & LPT)**.
+   #. Find the port named *JLink CDC UART Port* and note down the number in parentheses.
+
+      If you have more than one J-Link UART Port, unplug the one that you want to use, plug it back in, and observe which one appeared last.
+
+      Your DK can show up as two consecutive COM ports.
+      If this is the case, you need to test which COM port is the correct one.
+
+#. Configure the following settings in the **Configure the serial line** section:
+
+   * **Speed (baud):** 115200
+   * **Data bits:** 8
+   * **Stop bits:** 1
+   * **Parity:** None
+   * **Flow control:** RTS/CTS
+
```

Comment:
Instead of listing these settings, link to `test_and_optimize`, which lists this as reference: https://github.com/nrfconnect/sdk-nrf/pull/13663/files#diff-0714b3a4a71a5817b02b064cd9374dafb9e4db26327a36a62a1eb33b17e72f5a